### PR TITLE
Daniel Mc week 1 homework

### PR DESCRIPTION
**Pull request by danielmcmillan**

### CODE REVIEW

File: cis194/week1/daniel-mc/Homework.hs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+-- Exercise 2
+
+doubleEveryOtherRev :: [Integer] -> [Integer]
+doubleEveryOtherRev []             = []
+doubleEveryOtherRev [x           ] = [x]
+doubleEveryOtherRev (x1 : x2 : xs) = x1 : (x2 * 2) : doubleEveryOtherRev xs
+
+doubleEveryOther :: [Integer] -> [Integer]
+doubleEveryOther = reverse . doubleEveryOtherRev . reverse
```
Comment (stevemao) [inline cis194/week1/daniel-mc/Homework.hs]: I think @shineli1984 mentioned on my PR as well there's no need to `reverse` back

Comment (danielmcmillan) [reply cis194/week1/daniel-mc/Homework.hs]: Oh that's true since it's summed anyway, but then the tests for doubleEveryOther would fail

Comment (stevemao) [reply cis194/week1/daniel-mc/Homework.hs]: @shineli1984 I think that makes sense. Should we merge this?

Comment (shineli1984) [reply cis194/week1/daniel-mc/Homework.hs]: I think Daniel can merge it when he's happy with it. But it LGTM.

Comment (danielmcmillan) [reply cis194/week1/daniel-mc/Homework.hs]: Thanks yeah I'm happy with it. I don't think I have permission to merge

**Pull request merged**